build.gradle.kts

```diff
@@ -186,11 +186,21 @@ googleJavaFormat {
 tasks.check.configure {
   dependsOn(tasks.verifyGoogleJavaFormat)
 }
+tasks.checkstyleMain.configure {
+  // Set up a soft dependency so that verifyGoogleFormat suggests running googleJavaFormat,
+  // before devs start fixing individual checkstyle violations manually.
+  shouldRunAfter(tasks.verifyGoogleJavaFormat)
+}
+tasks.named("checkstyleIntegTest").configure {
+  // Set up a soft dependency so that verifyGoogleFormat suggests running googleJavaFormat,
+  // before devs start fixing individual checkstyle violations manually.
+  shouldRunAfter(tasks.verifyGoogleJavaFormat)
+}
```

Comment:
Nit: if I'm understanding the intention here correctly, perhaps something like this could also work?
```
tasks.withType<Checkstyle> {
  shouldRunAfter(tasks.verifyGoogleJavaFormat)
}
```

Reply:
https://github.com/TWiStErRob/gcp-gradle-plugin/commit/e09b7e0502cc309d3774dbd18c9205c60666b7c3

Reply:
Note: never ever ever ever use `withType<> {}` it's eager Gradle configuration (it'll create all tasks).

`withType<>().configureEach {}` is the lazy new API from [docs](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#sec:old_vs_new_configuration_api_overview):
![image](https://user-images.githubusercontent.com/2906988/216016773-54783a86-cc5a-41ae-95bd-a5196240566b.png)

Reply:
I'm planning on opening a PR eventually to make this plugin lazy(er).

Reply:
> Note: never ever ever ever use `withType<> {}` it's eager Gradle configuration (it'll create all tasks).

Ah, thanks for catching this and replacing with the alternative! TIL :)